.github/workflows/test.yml

```diff
@@ -0,0 +1,211 @@
+name: Test localgovdrupal/localgov_claro drupal-theme
+
+on:
+  push:
+    branches:
+      - '1.x'
+  pull_request:
+    branches:
+      - '1.x'
```

Comment:
The latest version of this module is tracking the 2.x branch, so these references to 1.x should be updated to 2.x and the tests will run.